deployment/services/public-graphql-api-gateway.ts

```diff
@@ -69,6 +76,13 @@ export function deployPublicGraphQLAPIGateway(args: {
         ),
         SUPERGRAPH_ENDPOINT: supergraphEndpoint,
         OPENTELEMETRY_COLLECTOR_ENDPOINT: args.observability.tracingEndpoint ?? '',
+
+        // Hive Console OTEL Tracing configuration
+        HIVE_HIVE_TRACE_ENDPOINT: serviceLocalEndpoint(args.otelCollector.service).apply(
```

Comment:
Why double: `HIVE_HIVE_` ?

Reply:
It's actually only on `HIVE_TRACE_ACCESS_TOKEN`, so `HIVE_TRACE_ENDPOINT` is fine. But for coherence, probably better to have the same prefix everywhere :-P